independent-publisher-connectors/Anthropic/apiDefinition.swagger.json

```diff
@@ -139,7 +149,205 @@
               ]
             }
           }
-        ]
+        ],
+        "x-ms-openai-data": {
+          "openai-enabled": true,
+          "operations": [
+            {
+              "operationId": "ClaudePost",
+              "x-ms-require-user-confirmation": true
+            }
+          ]
+        }
+      }
+    },
+    "/v1/messages": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "completion": {
+                  "type": "string",
+                  "description": "",
+                  "title": "completion"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "",
+                  "title": "id"
+                },
+                "model": {
+                  "type": "string",
+                  "description": "",
+                  "title": "model"
+                },
+                "stop_reason": {
+                  "type": "string",
+                  "description": "",
+                  "title": "stop_reason"
+                },
+                "type": {
+                  "type": "string",
+                  "description": "",
+                  "title": "type"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create a message",
+        "description": "Send a prompt to Claude to return a message.",
+        "operationId": "MessagePost",
+        "parameters": [
+          {
+            "name": "anthropic-version",
+            "in": "header",
+            "required": true,

```

Comment:
The required parameter with default value has been added. ValuePath: operation/MessagePost/parameter/anthropic-version. Can you please confirm that existing flows will not be broken?